Detectors/ITSMFT/ITS/reconstruction/src/CATracker.cxx

```diff
@@ -430,8 +430,8 @@ void Tracker::MakeCells(int iteration) {
       if (tLUT[iD][idx] == -1) continue;
       for (size_t c1 = tLUT[iD][idx]; c1 < mCells[iD + 1].size(); ++c1) {
         if (idx != mCells[iD + 1][c1].d0()) break;
-        float* n0 = mCells[iD][c0].GetN();
-        float* n1 = mCells[iD + 1][c1].GetN();
+        auto& n0 = mCells[iD][c0].GetN();
```

Comment:
why not `const`? I suspect you can also avoid the reference with modern compilers, then, since the code is inline and the compiler should avoid the intermediate copy (to be checked, of course).